### ISSUE TITLE
No information displayed for one of my systems, unicode error

### ISSUE THREAD

**Issue by af7567** [dendory/Healthstone#1]
Hi, I was just trying Healthstone out to monitor a couple of servers and found that one of them doesn't display any info or graph if I click the status.

The dashboard is running on Linux with apache 2.4.20 and python 3.5.2.  The error shown in the apache logs is
```
AH01215: Traceback (most recent call last):: /var/www/example.com/healthstone/dashboard.py, referer: https://example.com/healthstone/
AH01215:   File "/var/www/example.com/healthstone/dashboard.py", line 222, in <module>: /var/www/example.com/healthstone/dashboard.py, referer: https://example.com/healthstone/
AH01215:     print("<div class='panel-heading'><h3 class='panel-title'><span style='float:right'><i>" + time.strftime("%Y/%m/%d %H:%M:%S", time.localtime(row[6])) + "</i></span>" + row[1] + " (" + row[0] + ")</h3></div><div class='panel-body'><h4>System Profile</h4><pre>" + row[5] + "</pre><br>"): /var/www/example.com/healthstone/dashboard.py, referer: https://example.com/healthstone/
AH01215: UnicodeEncodeError: 'ascii' codec can't encode character '\\xae' in position 248: ordinal not in range(128): /var/www/example.com/healthstone/dashboard.py, referer: https://example.com/healthstone/
```
The 2 systems are a Windows server 2008 R2 which shows up fine, and a Windows SBS2011 server which doesn't.

The problem is caused by the (R) symbol in its description "Microsoft Windows® Small Business Server 2011 Standard  (64-bit)"

On a slightly unrelated note, Healthstone doesn't seem to be able to get windows update info on SBS2011 either
```
--> [CheckUpdates] Registry query failure: System.NullReferenceException: Object reference not set to an instance of an object.
   at Healthstone.Program.ParseSections(Object sender, ElapsedEventArgs hse)
[CheckLUsers] Local users: 
```

**Comment by dendory** [comment]
The first is a Python issue. You need to tell Apache to use unicode with Python scripts. Try adding this to your Apache config:

    AddDefaultCharset UTF-8
    SetEnv PYTHONIOENCODING utf8

The second issue is now fixed in 2.0.1

**Comment by af7567** [comment]
Thanks!
I added those 2 lines to the .htaccess file in the healthstone/www directory and it is fixed.
